apisix/plugins/opa.lua

```diff
@@ -20,6 +20,7 @@ local http   = require("resty.http")
 local helper = require("apisix.plugins.opa.helper")
 local type   = type
 
+local pairs = pairs
```

Comment:
Why not put this under this line `local type = type`

Reply:
And a blank is needed before the line 23

Reply:
done